app/views/manage_assessments/coverages/new.html.erb

```diff
@@ -23,5 +23,12 @@
       required: fields.options[:child_index] == 0 %>
   <% end %>
 
+  <div class="nested-form-actions">
+    <%= link_to_add_association t('.add_outcome'),
+      form,
+      :outcome_coverages,
```

Comment:
Cocoon is magic. 😄  It took me a while to figure out how it knew to render `app/views/manage_assessments/coverages/_outcome_coverage_fields.html.erb`.